astropy/units/tests/test_physical.py

```diff
@@ -503,3 +504,15 @@ def test_missing_physical_type_attribute():
     """
     with pytest.raises(AttributeError):
         length.not_the_name_of_a_str_or_physical_type_attribute
+
+
+@pytest.mark.parametrize('ptype', [
+    u.get_physical_type('length'),
+    u.get_physical_type('speed'),
+    u.get_physical_type('entropy')
+])
+def test_pickling(ptype):
+    # Regression test for #11685
+    pkl = pickle.dumps(ptype)
+    other = pickle.loads(pkl)
+    assert other == ptype
```

Comment:
```suggestion
@pytest.mark.parametrize('ptype_name', ['length', 'speed', 'entropy'])
def test_pickling(ptype_name):
    # Regression test for #11685
    ptype = u.get_physical_type(ptype_name)
    pkl = pickle.dumps(ptype)
    other = pickle.loads(pkl)
    assert other == ptype
```
Thank you for adding this test!  In retrospect, I should have added this in #11204.  Here's an optional suggestion that could make it a tad more straightforward to parse.

Reply:
Oh yea - that looks better :)